RDVFSharp/Commands/General/Ready.cs

```diff
@@ -8,65 +8,87 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using System.Timers;
 
 namespace RDVFSharp.Commands
 {
     public class Ready : BaseCommand<RDVFPlugin>
     {
+        public Timer ReadyTimer = new Timer(300000);
         public override string Description => "Sets a player as ready.";
 
         public override async Task ExecuteCommand(string character, IEnumerable<string> args, string channel)
         {
-            if (Plugin.GetCurrentBattlefield(channel).IsInProgress)
+            if (channel == "ADH-a823a4e998a2b3d31794")
```

Comment:
Store this in a "public static string RDVF_BAR_ROOMID" just so it's clearer as to what it's pointing